adafruit_requests.py

```diff
@@ -366,6 +360,60 @@ def __init__(
         self._session_id = session_id
         self._last_response = None
 
+    def _build_boundary_data(self, files: dict):
```

Comment:
this returns an array of text and the files, this way the files themselves aren't loaded into memory until they are sent